sourcegraph/about-replatform
# Add blog index

Closes #58 by adding the blog index and new load more logic (to avoid the dreaded infinite scroll).

Also removes links to Liveblogs, related types etc.

[Spreadsheet](https://docs.google.com/spreadsheets/d/1A0MMGDNY9dR8soWLLFEyZyS-pwc26N9N_PYTWfldurk/edit?usp=sharing) outlining which posts have descriptions and which do not (as of 4/26/2022)

## To Test
- Visit `/blog` and make sure loading logic works correctly. 
- Test static builds
